app/src/main/java/com/android/voyageur/ui/search/SearchScreen.kt

```diff
@@ -397,7 +397,7 @@ fun SearchScreen(
                   }
                 }
           } else {
-            DiscoverContent(tripsViewModel, userViewModel)
+            DiscoverContent(searchQuery.text, tripsViewModel, userViewModel)
```

Comment:
Also, in this file at line 263, can you add the placeholder for the search bar for this screen to be "Search trips..."?